docs/releasenotes/0.22.md

```diff
@@ -0,0 +1,54 @@
+# actions-runner-controller v0.22.0
+
+This version of ARC focuses on scalability and reliablity of runners.
+
+## GitHub API Cache
+
+In terms of scalability, ARC now caches GitHub API responses according to their recommendation(=Cache-Control header).
```

Comment:
do we want to link to the doc https://docs.github.com/en/rest/overview/resources-in-the-rest-api#conditional-requests?
ex: [Cache-Control header](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#conditional-requests)

Reply:
Thanks! Added in [e402b2e](https://github.com/actions-runner-controller/actions-runner-controller/pull/1199/commits/e402b2e8fd0e32598e326dbad8a3cb3d66401a5e)